src/iso15118/io/sdp_server.cpp

```diff
@@ -138,7 +138,7 @@ TlsKeyLoggingServer::TlsKeyLoggingServer(const std::string& interface_name, uint
     auto could_bind = false;
     auto source_port = 49152;
     for (; source_port < 65535; source_port++) {
-        sockaddr_in6 source_address = {AF_INET6, htons(source_port)};
```

Comment:
was this by accident, that `source_port` got replaced by `port`?

Reply:
Yes, thx for pointing that out :+1:

Reply:
Done